app/assets/javascripts/views/query_view.js.coffee

```diff
@@ -31,17 +31,17 @@ class QueryButtonsView extends Thorax.View
           , 3000
   initialize: ->
     @currentPopulation ?= 'IPP'
-  ipp: -> @model.ipp()
-  numerator: -> @model.numerator()
-  denominator: -> @model.denominator()
+  ipp: -> PopHealth.Helpers.formatNumber(@model.ipp())
```

Comment:
IMHO, I'd prefer to keep these as numbers and then do the specific formatting in the handlebars template with Numeral.js